runtime/src/chpl-tasks.c

```diff
@@ -37,17 +37,29 @@
 #include <sys/resource.h>
 #include <unistd.h>
 
+void chpl_task_warnNumThreadsPerLocale(const char* msg)
+{
+  static int quiet = -1;
+
+  if (quiet == -1) {
+    quiet = chpl_env_rt_get_bool("NUM_THREADS_PER_LOCALE_QUIET", 0);
+  }
```

Comment:
I don't think it's worth caching this. I'd probably just do 
```c
if (chpl_env_rt_get_bool("NUM_THREADS_PER_LOCALE_QUIET", false)) {
    chpl_warning(msg, 0, 0);
}
```

I'm not against caching it if you want to, I mostly just noticed it because of the mixed int/bool nature here (which is legal in C, but just looked a little odd to have something that's logically a bool with 3 possible values)

Reply:
Updated, caching was overboard